packages/apex-parser-ast/src/symbols/ApexSymbolManager.ts

```diff
@@ -137,6 +139,7 @@ export class ApexSymbolManager implements ISymbolManager, SymbolProvider {
 
   constructor() {
     this.symbolGraph = new ApexSymbolGraph();
+    ApexSymbolGraph.setInstance(this.symbolGraph);
```

Comment:
Leaving the comment here to say for the existing hover issue, I think something around here will likely be where the fix is, either in regeneration of this class to hydrate with the resolved symbolTable or sending in the updated table to be referenced when looking for the target symbol.